### ISSUE TITLE
Unable to generate Server token in ArcGIS_Server_Service

### ISSUE THREAD

**Issue by PleaseStopAsking** [Esri/arcgis-powershell-dsc#154]
When using the ArcGIS_Server_Service resource, it fails with the below error. This is due to the attempt to pass just the username of the $PublisherAccount parameter https://github.com/Esri/arcgis-powershell-dsc/blob/b191e088660eda8516f13bb15fbbc5a35afcfefd/Modules/ArcGIS/DSCResources/ArcGIS_Server_Service/ArcGIS_Server_Service.psm1#L111 while the Get-ServerToken function expects a $PSCredential object.

This only affects the Test method.

```
Cannot process argument transformation on parameter 'Credential'. PowerShell Desired State Configuration 
does not support execution of commands in an interactive mode. Please ensure that the underlying command is 
not prompting for user input, such as missing mandatory parameter, confirmation prompt etc.
    + CategoryInfo          : InvalidData: (:) [], CimException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Get-ServerToken
    + PSComputerName        : localhost
```

**Comment by shailesh91** [comment]
@PleaseStopAsking can you please submit this as part of the PR #156. Thanks.

**Comment by PleaseStopAsking** [comment]
@shailesh91 Done

**Comment by shailesh91** [comment]
Merged #156
